modules/cluster-logging-elasticsearch-status-comp.adoc

```diff
@@ -78,7 +82,7 @@ Containers:
   proxy:
     Container ID:  cri-o://3f77032abaddbb1652c116278652908dc01860320b8a4e741d06894b2f8f9aa1
     State:          Running
-      Started:      Mon, 08 Apr 2019 10:18:38 -0400
+      Started:      Mon, 08 Jun 2020 10:18:38 -0400
```

Comment:
registry.redhat.io/openshift4/ose-logging-elasticsearch5:v4.3   ->registry.redhat.io/openshift4/ose-logging-elasticsearch6@sha256:4265742c7cdd85359140e2d7d703e4311b6497eec7676957f455d6908e7b1c25

By the way, it is better to update digest after we kick-off the 4.5.

Reply:
Outdated